JS/validatecontact.js

```diff
@@ -0,0 +1,33 @@
+const contactForm = document.getElementById('contactMe');
+
+const emailInputValidator = /^[a-z0-9_.+-]+@[a-z-]+\.[a-z0-9-.]+$/;
+
+const onSubmit = (e) => {
+  e.preventDefault();
+  const message = document.getElementById('response');
+  const data = new FormData(e.target);
+  const email = data.get('email');
+
+  if (!emailInputValidator.test(email)) {
+    message.innerHTML = '* Error: Email field should be lower case';
+    message.style.color = '#602';
+    message.style.marginTop = '1rem';
+    return;
+  }
```

Comment:
- [ ] I noticed that your validation is not working as expected, because the form still submits despite the email containing uppercase characters. The expected behaviour is that the error message should show up close to the submit button or the input fields.
Please ensure that this works 👍